### DISCUSSION TITLE
High CPU loading in virtualbox

### DISCUSSION THREAD

**Discussion by aispeaking** [kovidgoyal/kitty#7296] [General]
The CPU loading is much higher than it in gnome-terminal when print message cyclically.
I guess it is because the virtualbox 6.1 3D acceleration do not support OpenGL3.x, is that right?

The virtuabox 7.0 seems upgraded it's 3D support, does any one have tried it? And how about vmware workstation?

I love kitty as it provides so much great features, but I don't have a linux host. I also do not care about the performance so much, a NORMAL is enough.

**Comment by Luflosi** [comment]
Try increasing these two values: https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.repaint_delay and https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.input_delay.
This will reduce the CPU usage at the cost of increased latency.
